### ISSUE TITLE
Unsupported relative calls

### ISSUE THREAD

**Issue by 0ca** [JonathanSalwan/Triton#227]
Hi, 

Triton is not translating correctly this kind of calls:
```
call qword ptr [rax+0x90]
```

Using the `tests/test_semantics.py` it can be seen the following output:
```
[KO] 0x7fffe207088f: call qword ptr [rax+0x90] (1 error)
     Register       : rip
     Symbolic Value : 00007fffe2070895
     Concrete Value : 00007fffe207bfb0
```
The symbolic value is the instruction just after the call.

**Comment by 0ca** [comment]
A program to reproduce the bug:
```
// gcc -masm=intel ./ir.c -o ir
#include<stdio.h>

void test(){  printf("Test\n");  }

int main(){
  int *p = test;
  asm("mov rax, %0" :: "r"(&p));
  asm("call [rax]");
  return 0;
}
```

**Comment by JonathanSalwan** [comment]
It's a design error with the `BaseIRBuilder::setup()` function. Fixed with the v0.3 (#226).
